Dockerfile

```diff
@@ -0,0 +1,17 @@
+FROM ruby:2.4
+
+RUN apt-get update
+RUN curl -sL https://deb.nodesource.com/setup_10.x | bash
+RUN apt-get update -qq && apt-get install -y build-essential nodejs git
+
+RUN mkdir /app
+WORKDIR /app
+
+COPY Gemfile /app
+COPY Gemfile.lock /app
+
+RUN gem install bundler -v 1.17.3
```

Comment:
why are we locking this to 1.17.3 the new bundler is faster and better

Reply:
updated the bundle using bundler version 2 in application and docker